docs/_docs/templates.md

```diff
@@ -421,56 +421,164 @@ The default is `default`. They are as follows (with what they filter):
+```
+
+You won't need to place your included content within the `_includes` directory. Instead,
+the inclusion is specifically relative to the file where the tag is being used. For example,
+if `_posts/2014-09-03-my-file.markdown` uses the `include_relative` tag, the included file
+must be within the `_posts` directory, or one of its subdirectories. 
+
+Note that you cannot use the `../` syntax to specify an include location that refers to a higher-level directory.
+
+All the other capabilities of the `include` tag are available to the `include_relative` tag,
+such as using variables.
+
+### Using variables names for the include file
+
+The name of the file you want to embed can be specified as a variable instead of an actual file name. For example, suppose you defined a variable in your page's frontmatter like this:
 
-You can also pass parameters to an include. Omit the quotation marks to send a variable's value. Liquid curly brackets should not be used here:
+```yaml
+---
+title: My page
+my_variable: footer_company_a.html
+---
+```
+
+You could then reference that variable in your include:
+
+```liquid
+{% raw %}{% include {{page.my_variable}} %}{% endraw %}
+```
 
+In this example, the include would insert the file footer_company_a.html from the \_includes directory.
+
+### Passing parameters to includes
+
+You can also pass parameters to an include. For example, suppose you have a file called note.html in your \_includes folder that contains this formatting: 
+ 
 ```liquid
-{% raw %}{% include footer.html param="value" variable-param=page.variable %}{% endraw %}
+{% raw %}<div markdown="span" class="alert alert-info" role="alert">
+<i class="fa fa-info-circle"></i> <b>Note:</b> {{include.content}}
+</div>{% endraw %}
 ```
 
-These parameters are available via Liquid in the include:
+The {% raw %}`{{include.content}}`{% endraw %} is a parameter gets populated when you call the include and specify a value for that parameter, like this:
 
 ```liquid
-{% raw %}{{ include.param }}{% endraw %}
+{% raw %}{% include note.html content="This is my sample note." %} {% endraw %}
 ```
 
-#### Including files relative to another file
+The value of `content` (which is `This is my sample note`) will be inserted into the {% raw %}`{{include.content}}`{% endraw %} parameter.
 
-You can also choose to include file fragments relative to the current file:
+Passing parameters to includes is especially helpful when you want to hide away complex formatting from your Markdown content. 
+
+For example, images with figure captions often have complicated formatting that you might want to simplify through an include with parameters. Here's an example:
+
+```html 
+<figure><a href="http://jekyllrb.com"><img src="logo.png" 
+style="max-width: 200px;" alt="Jekyll logo" /><figcaption>
+This is the Jekyll logo</figcaption></figure>
+```
+
+You could templatize this content in your include and make each value available as a parameter, like this:
```

Comment:
added a little disclaimer note. i didn't know that includes slow down the build time in a significant way. i thought it was mainly for loops.